MitchellGulledge/Meraki-vWAN
# Fix Issue #1

Fix issue #1.

## Description
Fixes issue #1 by adding payload and the correct header to the POST request in get_azure_virtual_wan_gateway_config.

## Related Issue
#1 

## Motivation and Context
Fixes the script from ending falsely

## How Has This Been Tested?
1. Create Azure environment with a WAN hub with effective routes
2. Create Function App
3. Confirm "No connected virtual networks or hubs to {virtual_wan_hub}" is not logged
